torch/testing/_internal/common_methods_invocations.py

```diff
@@ -13607,7 +13608,8 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                skipCUDAIf(not ((_get_torch_cuda_version() >= (11, 3))
                                or (_get_torch_rocm_version() >= (5, 2))),
                           "cusparseSDDMM was added in 11.2.1"),
-               skipCPUIfNoMklSparse, ],
+               skipCPUIfNoMklSparse,
+               skipXPU, ],
```

Comment:
```suggestion
               skipXPU],
```

Reply:
Hi, @EikanWang, seems most of the decorators augment in op_db has a ',' followed the last item. See [link1](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py#L10916) and [link2](https://github.com/pytorch/pytorch/blob/main/torch/testing/_internal/common_methods_invocations.py#L11787).